<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizSearchReplaceWidget</class>
 <widget class="QDialog" name="WizSearchReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find &amp; Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_repalce"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Replace:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_source"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_casesenitive">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_pre">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_replace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_replaceAll">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_source</tabstop>
  <tabstop>lineEdit_repalce</tabstop>
  <tabstop>checkBox_casesenitive</tabstop>
  <tabstop>btn_pre</tabstop>
  <tabstop>btn_next</tabstop>
  <tabstop>btn_replace</tabstop>
  <tabstop>btn_replaceAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
